pkg/kapp/clusterapply/waiting_changes.go

```diff
@@ -62,13 +65,19 @@ func (c *WaitingChanges) WaitForAny() ([]WaitingChange, error) {
 		for _, change := range c.trackedChanges {
 			change := change // copy
 
-			go func() {
+			go func(timeout time.Duration) {
```

Comment:
any specific reason why we are taking this in like this?

Reply:
Yeah we can remove this better to use it everywhere using `c.opts.ResourceTimeout`